cluster-autoscaler/go.mod

```diff
@@ -17,28 +17,27 @@ require (
 	github.com/google/go-querystring v1.0.0
 	github.com/jmespath/go-jmespath v0.0.0-20180206201540-c2b33e8439af
 	github.com/json-iterator/go v1.1.9
-	github.com/mesosphere/kommander-cluster-lifecycle/clientapis v0.0.0-20200323150606-ab6f57ecf9a3
+	github.com/mesosphere/kommander-cluster-lifecycle/clientapis v0.0.0-20200415132633-8e195ef03be9
 	github.com/mesosphere/konvoy/clientapis v0.0.0
 	github.com/prometheus/client_golang v1.1.0
 	github.com/satori/go.uuid v1.2.0
 	github.com/spf13/pflag v1.0.5
-	github.com/stretchr/testify v1.4.0
+	github.com/stretchr/testify v1.5.1
 	golang.org/x/crypto v0.0.0-20200220183623-bac4c82f6975
-	golang.org/x/oauth2 v0.0.0-20190604053449-0f29369cfe45
-	google.golang.org/api v0.9.0
+	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d
+	google.golang.org/api v0.15.1
 	gopkg.in/gcfg.v1 v1.2.0
 	gopkg.in/yaml.v2 v2.2.8
-	k8s.io/api v0.17.4
-	k8s.io/apimachinery v0.17.4
+	k8s.io/api v0.18.2
+	k8s.io/apimachinery v0.18.2
 	k8s.io/client-go v11.0.1-0.20190409021438-1a26190bd76a+incompatible
 	k8s.io/cloud-provider v0.16.4
-	k8s.io/component-base v0.17.3
+	k8s.io/component-base v0.17.4
```

Comment:
Why not bump this to be consistent with other `k8s.io/*` dependencies like https://github.com/mesosphere/autoscaler/pull/8/files#diff-5b1211f36242f6afe85bdb0062369dc3R31?

Reply:
@dkoshkin please address the comments if needed in a different PR.

Reply:
I just copied the new versions and ran `go mod tidy` ,  some transient deps bumped these versions.